features/step_definitions/division_steps.rb

```diff
@@ -0,0 +1,55 @@
+require 'calculator'
+
+When /^I divide two numbers$/ do
+	@quotient = Calculator.new.divide 4,2
+end
+
+Then /^I get the quotient$/ do
+	expect(@quotient).to eq 2
+end
+
+When /^I divide a number by zero$/ do
+	begin
+	Calculator.new.divide 4,0
+	rescue Exception => e

```

Comment:
This step does basically nothing. You had the right idea with a begin/rescue, but you needed to set the captured error to an instance variable and assert against that. This step should look like:

```ruby
When /^I divide a number by zero$/ do
  begin
    Calculator.new.divide 4, 0
  rescue => e
    @error = e
  end
end
```